Developer.md

```diff
@@ -0,0 +1,28 @@
+# Building and Testing the Java Client Core Library
+
+## Build/Dev Environment Prerequisites 
+ - [Java version 8+ e.g Temurin distribution](https://projects.eclipse.org/projects/adoptium.temurin/downloads)
+ - [Apache Maven version 3.8.5+](https://maven.apache.org/download.cgi)
+ - JUnit 5 Java Testing Framework version version 5.8+
```

Comment:
I don't know if this is necessary since the point of having the project as a Maven project is that all the dependencies are described by the POM file. JUnit 5 is just one of the dependencies.

Reply:
removed